inc/ticket.class.php

```diff
@@ -6944,10 +6944,11 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
       $tmp = array('tickets_id' => $this->getID());
       $fup             = new TicketFollowup;
       $ttask           = new TicketTask;
-      $doc             = new Document;
 
-      $canadd_fup      = $fup->can(-1, CREATE, $tmp);
-      $canadd_task     = $ttask->can(-1, CREATE, $tmp);
+      $canadd_fup      = $fup->can(-1, CREATE, $tmp) && !in_array($this->fields["status"],
```

Comment:
IMO, theses tests should be in canCreateItem methods of TicketFollowup & TicketTask

Reply:
Seems this is already tested in both `canCreateItem` methods; but with other checks as well

Reply:
Hum... I've read too quickly: only `getClosedStatusArray` is tested in those methods